manage_breast_screening/record_a_mammogram/views.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from django.forms import ModelForm
+from django.shortcuts import redirect, render
+
+from .forms import ScreeningAppointmentForm
+
+logger = logging.getLogger(__name__)
+
+
+def start_screening(request):
+    """
+    First step in the record a mammogram journey
+    """
+    # Placeholders
+    onward_step = "clinics:index"
+    stop_step = "clinics:index"
+
+    if request.method == "POST":
+        form = ScreeningAppointmentForm(request.POST)
+
+        if form.is_valid():
+            form.save()
+
+            # Go to the next step in the wizard
+            if form.cleaned_data["next_step"] == "continue":
+                return redirect(onward_step)
+            else:
+                return redirect(stop_step)
```

Comment:
ScreeningAppointmentForm could potentially decide what the next step is:


```suggestion
            return redirect(form.next_step)
```

Reply:
I think this would shadow the form field itself (form fields are attributes of the form) but I think I can do something similar to this with FormViews